sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/TaskResourceMapper.java

```diff
@@ -230,9 +230,11 @@ private static OfferEvaluationStage toEvaluationStage(
             Optional<String> resourceId,
             Optional<String> persistenceId) {
         if (resourceSpec instanceof NamedVIPSpec) {
-            return new NamedVIPEvaluationStage((NamedVIPSpec) resourceSpec, taskSpecName, resourceId);
+            NamedVIPSpec namedVIPSpec = (NamedVIPSpec) resourceSpec;
```

Comment:
@gabrielhartmann part of the reason that specs were formerly able to create their own evaluation stages polymorphically was to prevent brittleness like this, where each new case is going to require another `if` clause...